server/routes/functions.js

```diff
@@ -12,19 +12,14 @@ const registerUser = (req, res) => {
 			if (user) {
 				res.status(400).json({ Error: `${formData.username} already exists` })
 			} else {
-				db.createAccount(formData)
-					.then(() => {
-						res.status(200).json({ Success: `${formData.username} created` })
-					})
-					.catch(error => {
-						res.status(400).json({ error: error.message })
-						console.log('error', error.message)
-					})
+				return db.createAccount(formData)
 			}
 		})
+		.then(response => {
+			res.status(200).json({ Success: `${response.formData.username} created` })
+		})
 		.catch(error => {
-			res.status(500).json({ error: error.message }) // look into the right http status codes to return
-			console.log('error', error.message)
+			console.log('registerUser error: ', error.message)
```

Comment:
How will you handle these errors?

Reply:
Good question... I may have deleted this part on accident 😇

Reply:
Put it back in